src/store/application-state.js

```diff
@@ -11,7 +11,7 @@ const STATE = Vue.observable({
   download: false, // true if there is a downloaded that is waiting
   upload: false, // upload
   baseLayerId: null,
-  lng: 'en', // language default
+  language: 'en', // language default
```

Comment:
In order to avoid backwards compatibility issues we have also need to add here an alias between the old and the new property (ie. `ApplicationState.lng == ApplicationState.language` must be true at least until the next major release)

Reply:
@Raruto ok